[2022-07-04 15:58:25,918] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.create_table manual__2022-07-04T15:58:24.681331+00:00 [queued]>
[2022-07-04 15:58:25,927] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.create_table manual__2022-07-04T15:58:24.681331+00:00 [queued]>
[2022-07-04 15:58:25,928] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 15:58:25,928] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-04 15:58:25,928] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-04 15:58:25,949] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2022-07-04 15:58:24.681331+00:00
[2022-07-04 15:58:25,954] {standard_task_runner.py:52} INFO - Started process 645 to run task
[2022-07-04 15:58:25,970] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'create_table', 'manual__2022-07-04T15:58:24.681331+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpcyn6sywy', '--error-file', '/tmp/tmphqxncw8t']
[2022-07-04 15:58:25,971] {standard_task_runner.py:80} INFO - Job 23: Subtask create_table
[2022-07-04 15:58:26,051] {task_command.py:370} INFO - Running <TaskInstance: first_dag.create_table manual__2022-07-04T15:58:24.681331+00:00 [running]> on host b631230cc7b5
[2022-07-04 15:58:26,176] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-07-04T15:58:24.681331+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-04T15:58:24.681331+00:00
[2022-07-04 15:58:26,192] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-07-04 15:58:26,196] {dbapi.py:213} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );, parameters: None
[2022-07-04 15:58:26,197] {postgres.py:94} INFO - NOTICE:  relation "users" already exists, skipping

[2022-07-04 15:58:26,218] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=create_table, execution_date=20220704T155824, start_date=20220704T155825, end_date=20220704T155826
[2022-07-04 15:58:26,261] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-04 15:58:26,301] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
